src/Console/StartWebSocketServer.php

```diff
@@ -46,8 +48,21 @@ public function handle()
 
     protected function configureStatisticsLogger()
     {
+        if (config('websockets.statistics.dns_lookup')) {
+            $dnsConfig = DnsConfig::loadSystemConfigBlocking();
+
+            $dns = (new DnsFactory)->createCached(
+                $dnsConfig->nameservers
+                    ? reset($dnsConfig->nameservers)
+                    : '1.1.1.1',
+                $this->loop
+            );
+        } else {
```

Comment:
Avoid this else statement by refactoring to a dedicated method (that uses early returns) that 'll return a `DnsResolver`